### ISSUE TITLE
Access to unit list

### ISSUE THREAD

**Issue by phrozen** [gentooboontoo/js-quantities#48]
Hi, I'm using this awesome library for a project and although there exist a way to get the "well-known kinds" it is useless to me without having a way to obtain the unit list in that category (to put in a select widget or something), the code already has the list, it is just a matter of exposing it and maybe filtering it.

Thoughts?

**Comment by gentooboontoo** [comment]
Could something like what was proposed in PR #14 fit your need?

```javascript
getUnits("radiation") -> [["Gy","gray","grays"],["R","roentgen"], ...]
```

Each unit would be grouped by alias.

Side questions, would you display every alias or arbitrarily one of them? How will you filter out unneeded units like light years or astronomical units from `length` kind for instance?

**Comment by phrozen** [comment]
That would be great!! Is that commit on master?

I guess one way to select them is either take the SI canonical form or the full name (either m or meter, N or Netwon) I would have to check once I have the list via getUnits() and maybe select (Hopefully) alias[0].

I might not need to filter out some units, but in case I would, I would use a dictionary build after having access to the units by kind, and filter them like that. Thanks!

**Comment by lukelaws** [comment]
Hi there,
I have a need for this as well.  And using it in a select box is also my intention.  I read over PR #14 and I agree getting the available units of a particular kind is a separate concern from getting aliases.  Would it be acceptable to use the key names in UNITS as a single name for each?  This is working well for my application:  1b8fb5bf4f7d2455e4241a0c267d405a3caf385e

**Comment by gentooboontoo** [comment]
Fixed by #53
